docs/best-practice/limit-and-offset-vs-cursor-api-and-hasmore.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_position: 50
+title: LIMIT and OFFSET vs Cursor API and hasMore
+---
+
+Queries that return large volumes of data may require more than one response to provide the complete results. A common method is using the `LIMIT` and `OFFSET` statements in the query. However, this is not the optimized approach. When these options are used, each time the query is invoked query processing is performed. 
```

Comment:
Use "might" rather than "may" to refer to the possibility that something could happen. "May" suggests permission, as in, "You may submit that PR."